src/styles/index.scss

```diff
@@ -1,3 +1,127 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@700&display=swap");
+
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  background-color: #fff;
+  font-family: Roboto, Arial, sans-serif;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.card__img {
+  padding: 0 3px 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__name {
+  margin: 0;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  padding-bottom: 4px;
+}
+
+.card__code {
+  margin: 0;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  padding-bottom: 16px;
+}
+
+.card__rate {
+  justify-content: space-between;
+  padding-bottom: 24px;
+  align-items: flex-end;
+}
+
+.card__rate,
+.card__stars {
+  display: flex;
+}
+
+.card__reviews-number {
+  margin: 0;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #060b35;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  margin: 0;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+}
+
+.card__value {
+  margin: 0;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__buyButton {
+  color: #f3f3f3;
+  text-transform: uppercase;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  box-sizing: border-box;
+  margin-top: 16px;
+  height: 40px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border: 1px solid;
+  border-radius: 5px;
+  text-decoration: none;
+  background-color: #00acdc;
+}
+
+.card__buyButton:hover {
+  background-color: #f3f3f3;
+  color: #00acdc;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: 50%;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 .stars__star:first-child,
+.stars--2 .stars__star:nth-child(-n+2),
+.stars--3 .stars__star:nth-child(-n+3),
+.stars--4 .stars__star:nth-child(-n+4),
+.stars--5 .stars__star:nth-child(-n+5) {
+  background-image: url(../images/star-active.svg);
+}
```

Comment:
Good usage for nth-child pseudo-class
However, you can further improve this using  `@for`

https://sass-lang.com/documentation/at-rules/control/for/